cpp/tests/CMakeLists.txt

```diff
@@ -133,6 +133,15 @@ function(ConfigureTest CMAKE_TEST_NAME)
     endif(OpenMP_CXX_FOUND)
```

Comment:
The following targets need to be installed as well:
- targets made via ConfigureTestMG()
- targets made via ConfigureCTest()

Reply:
@robertmaynard, unless I'm mistaken, I think I've already covered those targets here. I just rebased on the latest changes and pushed a commit to update the `DESTINATION` and `RPATH` path for each target.